docs/getting-started-guides/logging.md

```diff
@@ -167,7 +167,7 @@ Here is some sample output:
 
 ![BigQuery](bigquery-logging.png)
 
-We could also fetch the logs from Google Cloud Storage buckets to our desktop or laptop and then search them locally. The following command fetches logs for the counter pod running in a cluster which is itself in a GCE project called `myproject`. Only logs for the date 2015-06-11 are fetched.
+We could also fetch the logs from Google Cloud Storage buckets to our desktop or laptop and then search them locally. The following command fetches logs for the counter pod running in a cluster which is itself in a Compute Engine project called `myproject`. Only logs for the date 2015-06-11 are fetched.
```

Comment:
hmm, possibly might be confusing, we might not need to be introduce the fact that projects span GCP. I read this as the project related to GCE, which we referred to earlier in the document: "...Google Compute Engine cluster...".